etna/records/views/choosers.py

```diff
@@ -23,7 +23,7 @@ class KongModelChooserMixinIn(ChooserMixin):
 
     def get_object_list(self, search_term=""):
         """Filter object list by user's search term"""
-        return self.model.search.filter(term=search_term, stream="evidential")
```

Comment:
Still passing empty, the chooser pop will fail on the initial load. In DF 52 I return an empty list if Not keyword.

Reply:
This is still hardcoded  'stream="evidential" ', I believe this is an enum now can we not use something like stream=Stream.EVIDENTIAL

Reply:
Thanks Mo.

I can understand the confusion as there are multiple streams of work being carried out in similar areas. This PR doesn't supersede the work in DF-52.

Good point on using the enum! No point in writing it if I'm not going to use it! 😄